strawberryfields/gbs/clique.py

```diff
@@ -22,13 +22,17 @@
 This module provides tools for users to identify large cliques in graphs. A clique is a subgraph
 where all nodes are connected to each other. The maximum clique problem is to identify the
 largest clique in a graph. The problem is NP-Hard in a worst-case setting, which is why it is
-valuable to develop better algorithms to tackle it.
+valuable to develop better algorithms to tackle it. It has been shown that samples from GBS can
+be used to select dense subgraphs as a starting seed for heuristic algorithms
+:cite:`banchi2019molecular`.
+
+An accompanying tutorial can be found :ref:`here <gbs-clique-tutorial>`.
 
 Heuristics

```

Comment:
```suggestion
Algorithm
```
I think it's not so useful to emphasize that it's a "heuristic" algorithm, it is arguably one of the best existing algorithms for max clique!